website/docs/layouts/app-frame/index.md

```diff
@@ -15,5 +15,6 @@ navigation:
 </section>
 
 <section data-tab="Version history">
+  @include "partials/version-history/4.15.0.md"
```

Comment:
thought: next is 4.14.0; unless you would rather have this published afterward/separately?
```suggestion
  @include "partials/version-history/4.15.0.md"
```

Reply:
I didn't know if and when it was going to be released, so I tried to be conservative. I'll update before merging, if approved.